_rules/role-required-states-and-properties-4e8ab6.md

```diff
@@ -60,47 +64,132 @@ This rule relies on browsers and assistive technologies to support leaving out [
 
 #### Passed Example 1
 
-This `checkbox` has the required property `aria-checked`.
+This `heading` has the required `aria-level` property.
 
 ```html
-<div role="checkbox" aria-checked="false"></div>
+<div role="heading" aria-level="1">
+	My First Heading
+</div>
 ```
 
 #### Passed Example 2
 
+This `checkbox` has the required `aria-checked` property.
+
+```html
+<div role="checkbox" aria-checked="false"></div>
+<div id="label">Check me</div>
+```
+
+#### Passed Example 3
+
 This `scrollbar` has the required properties `aria-controls` and `aria-valuenow`. `aria-valuemin` has a default value of 0 and `aria-valuemax` of 100.
 
 ```html
 <div role="scrollbar" aria-controls="content" aria-valuenow="0"></div>
 <main id="content"></main>
 ```
 
-#### Passed Example 3
+### Passed Example 4
+
+These `option` nodes do not need the required `aria-selected` property because it has a default value of `false`.
+
+```html
+<div id="label">Tags</div>
+<ul role="listbox" aria-labelledby="label">
+	<li role="option">Zebra</li>
+	<li role="option">Zoom</li>
+</ul>
+```
+
+### Passed Example 5
+
+This `separator` is not a `widget` because it is not [focusable][]. The `separator` role only requires the `aria-valuenow` property when the element is focusable.
+
+```html
+<p>My first HTML</p>
+<div role="separator"></div>
+<p>My last HTML</p>
+```
+
+#### Passed Example 6
 
-This `combobox` has required properties `aria-controls` and `aria-expanded`. `aria-controls` references an element that does not exist, but may be added to the page when expanded.
+This `combobox` has the required properties `aria-controls` and `aria-expanded`.
 
 ```html
-<div role="combobox" aria-controls="someElementId" aria-expanded="false"></div>
+<label for="tag_combo">Tag</label>
+<input type="text" id="tag_combo" role="combobox" aria-expanded="true" aria-controls="popup_listbox" />
+<ul role="listbox" id="popup_listbox">
+	<li role="option">Zebra</li>
+	<li role="option" id="selected_option">Zoom</li>
+</ul>
 ```
 
 ### Failed
 
 #### Failed Example 1
 
-This `combobox` is missing the required `aria-controls` property.
-
-**Note**: In [WAI-ARIA 1.2][], `combobox` will also require `aria-expanded`.
+This `heading` does not have the required `aria-level` property. Prior to [WAI-ARIA 1.2][] the `heading` role had an implicit default `aria-level` value of `2`. As of WAI-ARIA 1.2 this property must be explicitly set.
```

Comment:
I'm not sure about putting "historical" details in the descriptions 🤔 Maybe just saying that the property has no default (no need to say how it was in the previous versions, people can look at them on their own).
```suggestion
This `heading` does not have the required `aria-level` property, which has no default value.
```